src/payment/models/pricing/services/dfi-pricing-dex.service.ts

```diff
@@ -18,26 +19,28 @@ export class DfiPricingDexService implements PriceProvider {
   }
 
   async getPrice(from: string, to: string): Promise<Price> {
-    if (to !== 'DFI') {
-      throw new Error(`DfiPricingDexService supports only DFI as target asset, instead provided: ${to}`);
-    }
+    const fromAsset = await this.assetService.getAssetByQuery({
+      dexName: from,
+      blockchain: Blockchain.DEFICHAIN,
+      type: AssetType.TOKEN,
+    });
 
-    const dfi = await this.assetService.getAssetByQuery({ dexName: 'DFI', blockchain: Blockchain.DEFICHAIN });
+    const toAsset = await this.assetService.getAssetByQuery({
+      dexName: to,
+      blockchain: Blockchain.DEFICHAIN,
+      type: AssetType.TOKEN,
+    });
 
     const liquidityRequest: LiquidityRequest = {
       context: LiquidityOrderContext.PRICING,
       correlationId: uuid(),
-      referenceAsset: from,
+      referenceAsset: fromAsset,
       referenceAmount: 0.001,
```

Comment:
Should we use 0.001 in case of BTC and 1 else? As we have it at another place?

Reply:
yes, kets keep it safe, will change to condition